### ISSUE TITLE
_restart plugin improvement

### ISSUE THREAD

**Issue by trollollol123** [brutalcs/BrutalCS#4]
We are using a plugin to restart the server every night. Now this plugin wont just restart the server at set time, it will wait until the server is empty. 

The problem is that sometimes a server never actually gets empty, and then the server wont restart and it will start to perform worse over time.

It would be nice if the plugin could keep track if it did restart after the set time, and if not, it could force an restart at a second specified time with some warning in the chat before it goes down. like when its 5 min left and when its 30 seconds left or something.

**For example:**
DM #3 is set to restart at 5am. But since its a popular server, the players stay on it all night and and it wont actually have a chance to restart again until 5am next morning. So with this new idea, the server would force a restart at maybe 2pm and then again at 5am if the server is empty. :p

sm_autorestart_time
sm_autorestart_backup_time (or whatever :P)

https://forums.alliedmods.net/showthread.php?p=777168

**Comment by B3none** [comment]
Perhaps just force a restart at 5am whether there is players or not? 👍

**Comment by trollollol123** [comment]
> Perhaps just force a restart at 5am whether there is players or not? 👍

No, then we loose the players for next 6 hours or so. They dont join back at that time if the server goes down. :P

**Comment by B3none** [comment]
Okie doke, I'll make the edits now
